packages/core/src/__integrationtests__/Deposit.spec.ts

```diff
@@ -9,111 +9,87 @@
  * @group integration/deposit
  */
 
-import {
-  createLightDidFromSeed,
-  createOnChainDidFromSeed,
-  DemoKeystore,
-  DidChain,
-  FullDidDetails,
-  SigningAlgorithms,
-} from '@kiltprotocol/did'
+import { DemoKeystore, DidChain, FullDidDetails } from '@kiltprotocol/did'
 import {
   IRequestForAttestation,
-  KeyRelationship,
   KeyringPair,
-  KeystoreSigner,
   SubmittableExtrinsic,
 } from '@kiltprotocol/types'
 import { DecoderUtils, Keyring } from '@kiltprotocol/utils'
 import { BlockchainUtils } from '@kiltprotocol/chain-helpers'
-import { mnemonicGenerate, randomAsHex } from '@polkadot/util-crypto'
+import { mnemonicGenerate } from '@polkadot/util-crypto'
 import { BN } from '@polkadot/util'
 import {
-  createMinimalFullDidFromLightDid,
-  WS_ADDRESS,
+  createFullDidFromLightDid,
+  createFullDidFromSeed,
+  createMinimalLightDidFromSeed,
   devFaucet,
-  DriversLicense,
+  driversLicenseCTypeForDeposit as driversLicenseCType,
```

Comment:
Apparently there was a concurrency issue only when the tests are running in the CI ("it works on my machine"), for which I had to create a separate CType and use that throughout the deposit claiming tests.